src/flux/store.js

```diff
@@ -610,6 +622,10 @@ class Store extends EventEmitter {
   getIndexedLog = () => {
     return _store.logIndex;
   };
+
+  getStoreCopy = () => {
+    return _.cloneDeep(_store);
```

Comment:
Just `JSON.stringify` would have worked as well. 
But maybe `_.cloneDeep` is actually faster.
Not that this would matter.